lab-adan/lib/storage.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+let uuid = require('uuid');
+let fs = require('fs-extra');
+
+let storage = module.exports = {};
+
+storage.setItem = data => {
+  data.id = uuid.v1();
```

Comment:
Creating another ID for the resource here will change any value of the object that you're then setting to storage. This will definitely cause your tests to break if you are trying to supply an ID and then testing it.  Come find me this afternoon if you have questions about this.